README.md

```diff
@@ -30,16 +30,23 @@ spin up
 ## Building and Running
 
 ### Prerequisites
+__Note__: When using the devcontainer for development, the following dependencies are already installed for you...
 
 - [WASI SDK](https://github.com/WebAssembly/wasi-sdk) v16 or later, installed in /opt/wasi-sdk
 - [CPython](https://github.com/python/cpython) build prereqs (e.g. Make, Clang, etc.)
 - [Rust](https://rustup.rs/) (including `wasm32-wasi` target)
 - [Spin](https://github.com/fermyon/spin)
 - [pipenv](https://pypi.org/project/pipenv/) for installing Python project dependencies
 
+
 ### Instructions
 
-First, build CPython for wasm32-wasi.
+First, perform a git submodule update to update the cpython submodule
```

Comment:
Could this and the CPython build be part of the container creation?

Reply:
We could actually do a couple of things here. I flipped back and forth a couple of times on it previously. It all comes down to the usual workflow...if there is rarely a need to rebuild cpython or make changes to the branch etc, then it probably makes sense to move it into the dockerfile and take advantage of image caching...the build is already long enough to be a "walk away for a bit" build the first time, so I'm not adverse to just including the python compile as well.

Ultimately I made the decision to leave it this way after looking at how the development steps are listed in the README...ie: install these dependencies (stuff I put in dockerfile) and then steps begin with "run this script" which I left up to the user. 

I'll defer to @dicej and the guys at Fermyon though as far as which way would be a better developer experience given the normal workflow and tasks that are often performed?

Reply:
As discussed on discord, I switch the devcontainer to use an artifact compiled version of wasi cpython, so running these commands is no longer necessary when using it. Added notes to the affected steps in the README.